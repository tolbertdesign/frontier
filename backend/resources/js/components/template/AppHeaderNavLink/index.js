import AppHeaderNavLink from './AppHeaderNavLink.vue'

export default AppHeaderNavLink
