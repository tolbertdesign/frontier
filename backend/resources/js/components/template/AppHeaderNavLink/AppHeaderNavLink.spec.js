import { shallowMount, createLocalVue } from '@vue/test-utils'
import AppHeaderNavLink from './AppHeaderNavLink.vue'

const localVue = createLocalVue()

describe('AppHeaderNavLink', () => {
  it('Snapshop test for AppHeaderNavLink', () => {
    const wrapper = shallowMount(AppHeaderNavLink, {
      localVue,
      slots: {
        default: '<div class="slot"></div>',
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
