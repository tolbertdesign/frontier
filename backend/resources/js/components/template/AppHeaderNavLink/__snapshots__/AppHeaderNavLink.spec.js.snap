// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeaderNavLink Snapshop test for AppHeaderNavLink 1`] = `
<a class="m-0 p-4 px-4 font-medium text-gray-dark border-b border-grey-light hover:text-blue-darker hover:bg-grey-lighter">
  <div class="slot"></div>
</a>
`;
